### PR TITLE
mypy 0.720 & strict types

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: cwltool/main.py [hunk 1]
```diff
@@ -795,10 +801,13 @@ def my_represent_none(self, data):  # pylint: disable=unused-argument
...
-                executor = MultithreadedJobExecutor()
-                runtimeContext.select_resources = executor.select_resources
+                executor3 = MultithreadedJobExecutor()
+                runtimeContext.select_resources = executor3.select_resources
+                executor2 = executor3 # type: JobExecutor
             else:
-                executor = SingleJobExecutor()
+                executor2 = SingleJobExecutor()
+        else:
+            executor2 = executor
```
Comment (tetron) [inline cwltool/main.py]: executor / executor2 / executor3 is inelegant, can they get better names?

**Comment by tetron** [comment]
I gave this a once-over, it looks good to me.

**Comment by mr-c** [comment]
ooh, it broke some conformance tests.. https://ci.commonwl.org/job/cwltool-pr-conformance-multiver/2581/

**Comment by tetron** [comment]
that's what tests are for

**Pull request merged**